internal/service/federatedsettingsidentityprovider/resource_federated_settings_identity_provider_migration_test.go

```diff
@@ -1,13 +0,0 @@
-package federatedsettingsidentityprovider_test
-
-import (
-	"testing"
-
-	"github.com/mongodb/terraform-provider-mongodbatlas/internal/testutil/acc"
-	"github.com/mongodb/terraform-provider-mongodbatlas/internal/testutil/mig"
-)
-
-func TestMigFederatedSettingsIdentityProviderRS_basic(t *testing.T) {
```

Comment:
Would make sure once we implement full CRUD we add a migration test. cc: @oarbusi